app/components/elements/Table.js

```diff
@@ -72,6 +72,7 @@ const StyledTable = styled(Base)`
     color: inherit;
     font-size: inherit;
     font-family: inherit;
+    background-clip: padding-box
```

Comment:
```suggestion
    background-clip: padding-box;
```
Should always have a semicolon after style declarations.  No harm here, but any subsequent styles added would be ignored.